plugins/modules/clusters.py

```diff
@@ -63,7 +68,7 @@
 API_URL = f"https://api.openshift.com/api/assisted-install/{API_VERSION}"
 
 # add additional query parameters to the query_params_list
-QUERY_PARAMS_LIST = ["with_hosts"]
+QUERY_PARAMS_LIST = ["with_hosts","cluster_id"]
```

Comment:
The only query param possible for the endopoint you're using is `exclude-hosts`.  `cluster_id` is used as part of the endopoint. You can omit this one.

Reply:
I noticed in Swagger API spec that the supported query params for `/v2/clusters` are `get_unregistered_clusters openshift_cluster_id ams_subscription_ids with_hosts and owner`
For the `/v2/clusters/{cluster_id}` , the supported query params are `discovery_agent_version get_unregistered_clusters and exclude-hosts` . I guess there  is no use for `get_unregistered_clusters` when `cluster_id` is in the API path and could be a bug in the swagger spec. 
So, would the `get` be for single cluster i.e `/v2/clusters/{cluster_id}` and `list` be for `/v2/clusters/` ?
CC: @tonyskapunk